### PR TITLE
Adding falco dependecies to eliminate vulnerabilities

### PR DESCRIPTION
**Pull request by srishtih**
Fixes:

Related:

#### For new package PRs only

- [x] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [x] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Comment by srishtih** [comment]
Instead of using prebuilt falcotl and plugins, we are building them to eliminate the vulnerabilities stemming from these sources.

File: falco-plugins.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go-1.21
```
[the remark this answers is not in the document]
Comment (srishtih) [reply falco-plugins.yaml:15]: Not necessarily, will change it.

**Review by developer-guy — DISMISSED** [review]

File: falco.yaml [hunk 2]
```diff
@@ -26,6 +26,8 @@ environment:
       - cmake
       - curl-dev
       - elfutils-dev
+      - falco-plugins
```
Comment (mamccorm) [inline falco.yaml:29]: Are these meant to be build-time dependencies? I assumed they'd be runtime dependencies.

Or do we copy everything we need from these?

Comment (srishtih) [reply falco.yaml:29]: It makes sense to move them to runtime dependencies

**Review by dlorenc — APPROVED** [review]

**Pull request closed without merging**